packages/app/src/pages/sign-in/initial.tsx

```diff
@@ -82,7 +82,8 @@ export const SignIn: React.FC<SignInProps> = props => {
               placeholder="12-word Secret Key"
               as="textarea"
               value={seed}
-              autoCapitalize="false"
+              fontSize={['16px', '14px']}
```

Comment:
Won't this still zoom if you're on an iPad? Can we do 16px everywhere?

Reply:
Yep, changed to just 16px